hnn_core/optimization.py

```diff
@@ -468,12 +478,17 @@ def optimize_evoked(net, tstop, n_trials, target_dpl, initial_dpl, maxiter=50,
     which_drives: 'all' or list
         Evoked drives to optimize. If 'all', will opimize all evoked drives.
         If a subset list of evoked drives, will optimize only the evoked drives in the list.
+    return_rmse : bool
+        Returns list of unweighted RMSEs between data in dpl and exp_dpl 
+        for each optimization step
 
     Returns
     -------
     net : Network instance
         An instance of the Network object with the optimized configuration of
         attached drives.
+    opt_params : list
```

Comment:
```suggestion
    avg_rmse : list
```

Reply:
How about: 
iter_avg_rmse : list ?   avg_rmse is already used elsewhere for weighted rmse.

Reply:
works for me!